css/responsive.css

```diff
@@ -0,0 +1,103 @@
+@media (min-width: 1020px) {
+  .section {
+    padding-right: 10rem;
+    padding-left: 10rem;
+  }
+}
+
+@media (min-width: 768px) {
+  .mobile-only {
+    display: none;
```

Comment:
- Good work implementing the desktop version of your Portfolio. However, according to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.), Kindly make sure to replicate the Figma template as much as possible. Please change the font sizes for your navbar and greeting text of the desktop version to match the Figma design

**Figma Design**
![Screenshot from 2023-05-04 20-12-01](https://user-images.githubusercontent.com/97834160/236293462-053f8439-4448-4d19-b89b-c340909072e6.png)

**Your Design**
![Screenshot from 2023-05-04 20-02-45](https://user-images.githubusercontent.com/97834160/236293568-ce3763c4-1776-4279-8d0f-eee97d170b4f.png)